channels/constants.py

```diff
@@ -27,3 +29,14 @@
     COMMENTS_SORT_NEW,
     COMMENTS_SORT_OLD,
 )
+
+POST_TYPE = 'post'
+COMMENT_TYPE = 'comment'
+
+VoteActions = namedtuple('VoteActions', ['UPVOTE', 'DOWNVOTE', 'CLEAR_UPVOTE', 'CLEAR_DOWNVOTE'])
```

Comment:
Any reason not to use an enum for this since they're become nicer to work with in recent Python 3.x and we don't appear to care about the values and they don't cross celery broundaries or anything like that? This would probably be a little simpler:

```python
from enum import Enum, auto
class VoteActions(Enum):
    UPVOTE = auto()
    DOWNVOTE = auto()
    CLEAR_UPVOTE = auto()
    CLEAR_DOWNVOTE = auto()
```

Reply:
I completely forgot about Enum! Will do